src/super_gradients/training/transforms/keypoints/keypoints_compose.py

```diff
@@ -91,7 +91,7 @@ def _apply_transforms(cls, sample: PoseEstimationSample, transforms: List[Abstra
         """
         applied_transforms_so_far = []
         for t in transforms:
-            if t.additional_samples_count == 0:
+            if not hasattr(t, "additional_samples_count") or t.additional_samples_count == 0:
```

Comment:
I don't think its a good idea just to go on in case we did not load additional samples.
Suppose some user inherits from pose estimation dataset and does not load additional samples, they would have a hard time figuring out why Mosaic is not working.

Reply:
`t` is the transform, not the sample. So here I just check if the transform has the attribute `additional_samples_count`. If not, I assume its not a SG transform and therefore doesnt support the logic around `additional_samples_count`.

Also, here I am not changing the logic, it's just that albumentation transforms don't have attribute `additional_samples_count`, so this piece of code crashes. 

Does that make sens to you?